internal/cli/cmd/cluster/create.go

```diff
@@ -582,7 +582,7 @@ func (o *CreateOptions) buildDependenciesFn(cd *appsv1alpha1.ClusterDefinition,
 
 	// set component service account name
 	compSpec.ServiceAccountName = saNamePrefix + o.Name
-	o.shouldCreateDependencies = true
+	o.shouldCreateDependencies = false
```

Comment:
@ldming can we delete this dependency mechanism directly?

Reply:
yes，just remove it.